### DISCUSSION TITLE
I can't get teacher verification

### DISCUSSION THREAD

**Discussion by raiarruda** [community/community#53351] [GitHub Education]
The situation is as follows: I had two accounts - an older one under the handle @raiarruda and a newer one that I created when I began teaching under the handle @profaraissa. I had my @profaraissa account verified and was using the educational benefits package. However, I soon realized that it made no sense to have two profiles when I could have just one and create an organization to use with my students. I deleted the @profaraissa account and to my surprise, I was unable to verify my @raiarruda account. Despite submitting my payslip which displays my name and institution, and my email being automatically recognized as an educational institution, my application was immediately rejected. What steps should I take to resolve this issue?

**Comment by samus-aran** [comment]
Hi @raiarruda  👋

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

**Reply by raiarruda** [reply]
![image](https://user-images.githubusercontent.com/20960635/234551045-36bb5959-45fa-426d-b312-24a3202ddfa0.png)

**Reply by samus-aran** [reply]
Thanks @raiarruda 

If you have your profile information updated then I suspect that the text on the image you uploaded wasn't detected (no school name, or date). 

Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor rather than a phone perhaps.

We prefer students to submit a live photo of their evidence rather than an upload. We understand that students may not have access to high-quality cameras (laptop or phones) but as long as the text is legible the system should be able to process it. Here are a few tips on using a photo to capture.

1. Ensure your image (photo or upload) is legible. If you are unable to read its text on the image itself then the verification process may not be able to either. If you are taking a photo of a screen try to have it as large a resolution as possible - take a photo of the evidence on a monitor (a laptop perhaps - or a TV) rather than a phone.
2. If you can, print off your evidence and take a photo of this copy.
3. Keep the text legible. It's preferred to have the full shot of the evidence but try and capture the important sections of it to confirm your student status. If the photo is too large and makes the text tiny then zoom in appropriately.

Let us know here if you need further assistance
